app/ledger.js

```diff
@@ -995,7 +1007,9 @@ var getPaymentInfo = () => {
       info.address = client.getWalletAddress()
       if ((amount) && (currency)) {
         info = underscore.extend(info, { amount: amount, currency: currency })
-        if ((body.rates) && (body.rates[currency])) info.btc = (amount / body.rates[currency]).toFixed(8)
+        if ((body.rates) && (body.rates[currency])) {
+          info.btc = (amount / body.rates[currency]).toFixed(8)
+        }
```

Comment:
Whoops! I had added a line for `btcPrice` there, then removed it after you added the `transactions[n].contribution` field that made `btcPrice` unnecessary. But it looks like I didn't restore the old code style properly :-)